### PR TITLE
parse extracted values and show them in readonly inputs, parse addres…

### PR DESCRIPTION
**Pull request by lus21**
…s fields, state and adress names

### CODE REVIEW

File: utils/streetAbbreviations.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  WAY: ['WY', 'WAY'],
+  WAYS: ['WAYS'],
+  WELL: ['WELL', 'WL'],
+  WELLS: ['WELLS', 'WLS'],
+};
+
+const allStreetAbbreviations = flattenArray(Object.values(streetAbbreviations));
+export const allStreetAbbreviationsUnique = [...new Set(...allStreetAbbreviations)];
+
+export default allStreetAbbreviationsUnique;
```
Comment (lus21) [inline utils/streetAbbreviations.ts]: I didn't remove this because later we can use it for autocomplete I guess

File: utils/stateNames.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  'West Virginia': 'WV',
+  Wisconsin: 'WI',
+  Wyoming: 'WY',
+};
+
+const stateAbbreviations = Object.values(allStateNames);
+const fullStates = Object.keys(allStateNames);
+const fullStatesTypos = ['Ulah'];
+
+export const allStateVariations = flattenArray([...stateAbbreviations, ...fullStates, ...fullStatesTypos]);
```
Comment (lus21) [inline utils/stateNames.ts]: I didn't remove this because later we can use it for autocomplete I guess

File: global.d.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    type?: string;
+    city?: string;
+    state?: string;
+    zip?: string;
+    sec_unit_type?: string;
+    sec_unit_num?: string;
+  }
+
+  function parseLocation(addressToParse: string): ParseLocation;
+}
```
Comment (eryash12) [inline global.d.ts]: what is this?

Comment (eryash12) [reply global.d.ts]: is this declaring types for parse-address module

File: utils/flattenArray.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
+export const flattenArray = (arrays: any[]) => {
```
Comment (eryash12) [inline utils/flattenArray.ts]: why not just use lodash?

File: utils/stateNames.ts (hunk 2 above)
Comment (eryash12) [reply utils/stateNames.ts]: Lets remove we have the state mapping in informed-web

File: utils/streetAbbreviations.ts (hunk 1 above)
Comment (eryash12) [reply utils/streetAbbreviations.ts]: lets keep it in a branch

File: global.d.ts (hunk 3 above)
Comment (lus21) [reply global.d.ts]: yes, this is only declaring types for parse-address module because `parse-address` doesn't have types

**Pull request merged**